swagger/v1/swagger.yaml

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
I think the swagger directory will need to be in the `.gitignore` and remove this file since it looks like `swagger_helper.rb` generates this along with the other needed files to start the swagger server locally for hosting the docs.